app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.deuvox.deuvoxapp">
+    package="com.example.deuvox">
```

Comment:
ini knapa balik ke com.example? coba di pull dr master

Reply:
Okee sip

Reply:
Please pull it from sprint1-develop